spec/drift/migration_spec.cr

```diff
@@ -120,6 +120,58 @@ describe Drift::Migration do
         create_statement = migration.statements_for(:migrate).first
         create_statement.should eq(create_statement)
       end
+
+      it "handles statements that don't end with a semicolon" do
+        content = <<-SQL
+          -- drift:migrate
+          CREATE TABLE users (
+            id INTEGER PRIMARY KEY,
+            name TEXT NOT NULL
+          )
+          SQL
+
+        migration = Drift::Migration.from_io(content, 1)
+
+        migration.statements_for(:migrate).size.should eq(1)
+        migration.statements_for(:migrate).first.should contain("CREATE TABLE users")
+      end
```

Comment:
After some reflection, I don't think supporting this scenario is a good idea instead of a consistent treatment of semi-colons.